### DISCUSSION TITLE
API access

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#24] [New Feature Requests]
Hello

I've just released a new VSCode extension that should help security researchers writing their audit reports https://twitter.com/agfviggiano/status/1685349748847325184?s=46

In order to further improve the report description, proof of concept and recommendations, it would be great if we could search for similar issues on an comprehensive audit database before sending the prompt to ChatGPT. 

If Solodit had a public API, the extension could be turned into a frontend client that users would navigate when writing their reports, extracting the best justification, impact and remediation for common findings.

**Comment by hansfriese** [comment]
We are working on it!

**Reply by hans-cyfrin** [reply]
An update on this: we have other priorities that must be done first.
We will revisit this in the near future.

**Reply by ebaizel** [reply]
Hi, are there any updates or estimated timelines for this?
